lib/frontend/app.js

```diff
@@ -105,27 +103,30 @@ module.exports = (appTokenGenerator) => {
     res.redirect(info.external_url)
   })
 
-  const sentryMiddleware = async (err, req, res, next) => {
+  const addSentryContext = async (err, req, res, next) => {
     Sentry.withScope(async scope => {
-      scope.setTag('jiraHost', req.session.jiraHost || req.query.xdm_e)
-
-      if (req.session.githubToken) {
-        try {
-          const { data: { login } } = (await res.locals.github.users.get())
-          Sentry.setUser({ login })
-        } catch (err) {
-          req.log.error(err, 'unable to get user info')
-        }
+      const jiraHost = (req && req.session && req.session.jiraHost) || (req && req.query && req.query.xdm_e)
+      if (jiraHost) {
+        scope.setTag('jiraHost', jiraHost)
       }
 
       if (req.body) {
         Sentry.setContext('Body', req.body)
       }
-      Sentry.captureException(err)
```

Comment:
This is the bit that takes an exception we are in the process of sending to Sentry and tells Sentry to send it again.